changelog/_unreleased/2020-10-13-add-combination-list-to-available-combinations.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Add list of combinations to AvailableCombinationResult
+author: Joshua Behrens
+author_email: code@joshua-behrens.de
+author_github: @JoshuaBehrens
+---
+# Core
+* Add list of loaded combinations to return value of AvailableCombinationLoader
+___
+# Storefront
+* Add list of loaded combinations to return value of AvailableCombinationLoader
+___
```

Comment:
```suggestion
---
# Core
* Added list of loaded combinations to return value of AvailableCombinationLoader
___
# Storefront
* Added list of loaded combinations to return value of AvailableCombinationLoader
```

Reply:
Weird delta but I think just the tense has been changed